app/interfaces/Project.tsx

```diff
@@ -1,6 +1,7 @@
 export default interface Project {
```

Comment:
a note about types and interfaces: types overwrite any previous typing associated with the data so they should be used when typing primitives. Interfaces are best used when you want to extend the existing type of some data.

Reply:
Noted! I'll be swinging back to clean that up later. I had some bad luck trying to use types like this before, so I'll have to investigate why.